### PR TITLE
feat: Prometheus last day metrics

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

We can check how many metrics we accumulate from the instance stats API and from prometheus.

### CODE REVIEW

File: src/lib/metrics.ts [hunk 1]
```diff
@@ -235,6 +243,15 @@ export default class MetricsMonitor {
...
                 }
 
+                enabledMetricsBucketsPreviousDay.reset();
+                enabledMetricsBucketsPreviousDay.set(
+                    stats.previousDayMetricsBucketsCount.enabledCount,
+                );
+                variantMetricsBucketsPreviousDay.reset();
+                variantMetricsBucketsPreviousDay.set(
+                    stats.previousDayMetricsBucketsCount.variantCount,
+                );
```
Comment (FredrikOseberg) [inline src/lib/metrics.ts:253]: Does this need to be reset and set?

File: src/lib/features/instance-stats/instance-stats-service.ts [hunk 2]
```diff
@@ -244,6 +255,7 @@ export class InstanceStatsService {
             this.eventStore.filteredCount({ type: FEATURES_EXPORTED }),
             this.eventStore.filteredCount({ type: FEATURES_IMPORTED }),
             this.getProductionChanges(),
+            this.clientMetricsStore.countPreviousDayMetricsBuckets(),
```
Comment (FredrikOseberg) [inline src/lib/features/instance-stats/instance-stats-service.ts:258]: Does it make sense to have this in instance stats?

**Review by FredrikOseberg — COMMENTED** [review]
I'm wondering if this could be a static counter outside of instance stats, or if there is some reason we need to store it and reset and set it.

File: src/lib/metrics.ts (hunk 1 above)
Comment (kwasniew) [reply src/lib/metrics.ts:253]: I though that's what we do for all the other metrics. We need to update this one once per day. Other resets are not necessary. On the other hand it's just a count call under the hood.

File: src/lib/features/instance-stats/instance-stats-service.ts (hunk 2 above)
Comment (kwasniew) [reply src/lib/features/instance-stats/instance-stats-service.ts:258]: I wanted to have debug ability both in the API and prometheus. Do you have any example of a metric that is not either on instance stats or calculated after event emitter?

**Review by FredrikOseberg — APPROVED** [review]
LGTM

**Pull request merged**